script/test

```diff
@@ -230,6 +230,116 @@ do_cert_suite()
     fi
 }
 
+do_cert_bbr()
+{
+    if [[ ${THREAD_VERSION} != "1.2" ]]; then
+        echo "cert_bbr only work with THREAD_VERSION=1.2!"
+        exit 1
+    fi
+
+    if [[ ${VIRTUAL_TIME} != "0" ]]; then
+        echo "cert_bbr only work with VIRTUAL_TIME=0!"
+        exit 1
+    fi
+
+    export top_builddir="${OT_BUILDDIR}/cmake/openthread-simulation-1.2"
+    export top_builddir_1_2_bbr="${OT_BUILDDIR}/cmake/openthread-simulation-1.2-bbr"
+    export top_builddir_1_1="${OT_BUILDDIR}/cmake/openthread-simulation-1.1"
+    export PYTHONPATH=tests/scripts/thread-cert
+
+    python3 tests/scripts/thread-cert/run_bbr_tests.py --multiply "${MULTIPLY:-1}" "$@"
+}
+
+do_get_thread_wireshark()
+{
+    echo "Downloading thread-wireshark from https://github.com/openthread/wireshark/releases ..."
+    local download_url=https://github.com/openthread/wireshark/releases/download/ot-pktverify-20200727/thread-wireshark.tar.gz
+    local save_file=/tmp/thread-wireshark.tar.gz
+
+    rm -rf /tmp/thread-wireshark || true
+    rm -rf "${save_file}" || true
+    curl -L "${download_url}" -o "${save_file}"
+    tar -C /tmp -xvzf "${save_file}"
+
+    LD_LIBRARY_PATH=/tmp/thread-wireshark /tmp/thread-wireshark/tshark -v
+    LD_LIBRARY_PATH=/tmp/thread-wireshark /tmp/thread-wireshark/dumpcap -v
+    rm -rf "${save_file}"
+}
+
+do_build_otbr_docker()
+{
+    echo "Building OTBR Docker ..."
+    local otdir
+    local otbrdir
+    local otbr_options="-DOT_THREAD_VERSION=1.2 -DOT_BACKBONE_ROUTER=ON -DOT_DUA=ON -DOT_MLR=ON -DOT_REFERENCE_DEVICE=ON -DOT_SERVICE=ON -DOT_COVERAGE=ON"
+    local otbr_commit=${OTBR_COMMIT:-master}
+    local otbr_docker_image=${OTBR_DOCKER_IMAGE:-otbr-ot12-backbone-ci}
+
+    otbrdir=$(mktemp -d -t otbr_XXXXXX)
+    otdir=$(pwd)
+
+    (
+        cd "${otbrdir}"
+        git init
+        git remote add origin https://github.com/openthread/ot-br-posix.git
+        git fetch origin "${otbr_commit}" --depth 1
+        git reset --hard FETCH_HEAD
+        git submodule update --init --recursive --depth 1
+        rm -rf third_party/openthread/repo
+        cp -r "${otdir}" third_party/openthread/repo
+        rm -rf .git
+
+        cat >etc/docker/Dockerfile <<EOF
```

Comment:
Explain: we can't use the original Dockerfile of ot-br-posix because we need all build files in the `/app` directory.

Reply:
Using one Dockerfile is a good idea! We can introduce a new [ARG](https://docs.docker.com/engine/reference/builder/#arg) to the Dockerfile to build different Docker images. 
But since this requires submitting a PR to `ot-br-posix`, I would prefer to keep this PR unchanged and enhance it after the Dockerfile PR of `ot-br-posix` is merged. 

Thoughts?

Reply:
Submitted https://github.com/openthread/ot-br-posix/pull/556 for this purpose. But I hope we can review this PR without depending on https://github.com/openthread/ot-br-posix/pull/556.